manual/manual/library/stdlib.etex

```diff
@@ -99,6 +99,10 @@ be called from C \\
 "Spacetime" & p.~\pageref{Spacetime} & memory profiler \\
 "Sys" & p.~\pageref{Sys} & system interface \\
 \end{tabular}
+\subsubsection{Misc:}
+\begin{tabular}{lll}
+"Fun" & p.~\pageref{Fun} & Function values \\
```

Comment:
```suggestion
"Fun" & p.~\pageref{Fun} & function values \\
```
(or functional)

Reply:
It seems there's a mix of capital vs non-capitalized entries there but I can do this. 

Can a native english speaker please comment on "functional" vs "function" values, I hesitated quite a bit and the www didn't provide any answer.

Reply:
"Function" feels better here. "Functional" probably would confuse some people, though it wouldn't be wrong as such.